packages/snapshot/src/client.ts

```diff
@@ -32,6 +32,7 @@ interface AssertOptions {
   name?: string
   message?: string
   isInline?: boolean
+  isInsideEach?: boolean
```

Comment:
I don't think it should be part of the `packages/snapshot`. It couples too tightly with the runner.

Reply:
Make sense! I've removed it.